tests/static_map/unique_sequence_test.cu

```diff
@@ -242,17 +242,31 @@ __inline__ void test_unique_sequence(Map& map, size_type num_keys)
   }
 }
 
-TEMPLATE_TEST_CASE_SIG("Unique sequence",
-                       "",
-                       ((cuco::test::probe_sequence Probe, int CGSize), Probe, CGSize),
-                       (cuco::test::probe_sequence::double_hashing, 1),
-                       (cuco::test::probe_sequence::double_hashing, 2),
-                       (cuco::test::probe_sequence::linear_probing, 1),
-                       (cuco::test::probe_sequence::linear_probing, 2))
+TEMPLATE_TEST_CASE_SIG(
+  "Unique sequence",
+  "",
+  ((typename Key, typename Value, cuco::test::probe_sequence Probe, int CGSize),
+   Key,
+   Value,
+   Probe,
+   CGSize),
+  (int32_t, int32_t, cuco::test::probe_sequence::double_hashing, 1),
+  (int32_t, int64_t, cuco::test::probe_sequence::double_hashing, 1),
+  (int32_t, int32_t, cuco::test::probe_sequence::double_hashing, 2),
+  (int32_t, int64_t, cuco::test::probe_sequence::double_hashing, 2),
+  (int64_t, int32_t, cuco::test::probe_sequence::double_hashing, 1),
+  (int64_t, int64_t, cuco::test::probe_sequence::double_hashing, 1),
+  (int64_t, int32_t, cuco::test::probe_sequence::double_hashing, 2),
+  (int64_t, int64_t, cuco::test::probe_sequence::double_hashing, 2),
+  (int32_t, int32_t, cuco::test::probe_sequence::linear_probing, 1),
+  (int32_t, int64_t, cuco::test::probe_sequence::linear_probing, 1),
+  (int32_t, int32_t, cuco::test::probe_sequence::linear_probing, 2),
+  (int32_t, int64_t, cuco::test::probe_sequence::linear_probing, 2),
+  (int64_t, int32_t, cuco::test::probe_sequence::linear_probing, 1),
+  (int64_t, int64_t, cuco::test::probe_sequence::linear_probing, 1),
+  (int64_t, int32_t, cuco::test::probe_sequence::linear_probing, 2),
+  (int64_t, int64_t, cuco::test::probe_sequence::linear_probing, 2))
```

Comment:
This is ugly, they must provide something like a parameter axis.

Reply:
Guess who complained about this first? https://github.com/catchorg/Catch2/issues/1989